chromium_src/chrome/browser/ui/bookmarks/bookmark_utils.cc

```diff
@@ -69,6 +74,16 @@ bool ShouldShowAppsShortcutInBookmarkBar(Profile* profile) {
   return false;
 }
 
+std::u16string FormatBookmarkURLForDisplay(const GURL& url) {
```

Comment:
please add tests for this

Reply:
Added the test @bridiver Kindly review the code